docs/README.md

```diff
@@ -292,6 +292,7 @@ Check out the [**Product Summary documentation**](https://vtex.io/docs/component
 | `filtersTitleHtmlTag` | `string` | HTML tag for the filter's title. | `h5` |
 | `scrollToTop` | `enum` | Scrolls the page to the top (`auto` or `smooth`) or not (`none`) when selecting a facet. | `none` |
 | `truncateFilters` | `boolean` | Whether a filter selector with more than 10 filter options should shorten the list and display a `See more` button (`true`) or not (`false`). | `false` |
+| `priceRangeLayout` | `"default"` or `inputAndSlider` | whether a text field should appear to enter the price range (`"inputAndSlider"`), or not (`"default"`). | `"default"` |
```

Comment:
```suggestion
| `priceRangeLayout` | `enum` | Whether a text field to enter the desired price range should be displayed  (`inputAndSlider`) or not (`default`). | `default` |
```

Reply:
Wouldn't it be better if this prop was boolean? `true` for enabling the text field and `false` for not displaying it.

Reply:
We are facing problems related to boolean props. It turns out they are not future-proof 😞 .
For example, if someone creates another priceRangeLayout in the future, it will require a new prop that would conflict with this one.